### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

File: src/app/settings/emails/emails.component.html [hunk 1]
```diff
@@ -1,70 +1,69 @@
...
+            <div class="col-md-6 radio">
+                <div class="col-md-12">
+                    <input id="accountOnly" type="radio" name="pref" [(ngModel)]="profile.current.emailPreference" [value]="'accountOnly'">
+                    <label for="accountOnly" class="">Only receive account related emails, and those I subscribe to.</label>
+                    <span class="help-block">We'll only send you legal or administrative emails, and any emails you’ve specifically subscribed to.</span>
+                </div>
+            </div>
+            <button class="btn btn-default pull-right" type="submit">Save email preferences</button>
+        </form>
     </div>
```
Comment (joshuawilson) [inline src/app/settings/emails/emails.component.html]: you removed the closing div, please add it back

Comment (pmuir) [inline src/app/settings/emails/emails.component.html]: Fixed

File: src/app/settings/emails/emails.component.ts [hunk 2]
```diff
@@ -1,20 +1,43 @@
...
+
+  email: string;
+  keepPrivate: boolean;
 
   constructor(
-    private router: Router) {
+    private router: Router, public profile: ProfileService) {
+  }
+
+  delete(del: Email): void {
```
Comment (joshuawilson) [inline src/app/settings/emails/emails.component.ts]: 'delete' is a reserved word.
https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#221-reserved-words

Comment (pmuir) [inline src/app/settings/emails/emails.component.ts]: Fixed

**Pull request merged**